pep-0655.rst

```diff
@@ -3,12 +3,13 @@ Title: Marking individual TypedDict items as required or potentially-missing
 Author: David Foster <david at dafoster.net>
 Sponsor: Guido van Rossum <guido at python.org>
 Discussions-To: https://mail.python.org/archives/list/typing-sig@python.org/thread/53XVOD5ZUKJ263MWA6AUPEA6J7LBBLNV/
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 30-Jan-2021
 Python-Version: 3.11
 Post-History: 31-Jan-2021, 11-Feb-2021, 20-Feb-2021, 26-Feb-2021, 17-Jan-2022, 28-Jan-2022
+Resolution: https://mail.python.org/archives/list/python-dev@python.org/message/AJEDNVC3FXM5QXNNW5CR4UCT4KI5XVUE/
```

Comment:
fyi the typing-sig version is https://mail.python.org/archives/list/typing-sig@python.org/thread/AJEDNVC3FXM5QXNNW5CR4UCT4KI5XVUE/ - the emails on this one were caught in moderation so they didn't appear at the same time ;)  as discussion was on typing-sig, maybe link to that one?

Reply:
Well, the Typing-SIG thread is already linked in the `Discussions-To` header, and Python-Dev is the canonical place to post (non-Packaging, given its SC-delegated authority) PEP resolutions per [PEP 1](https://peps.python.org/pep-0001/#pep-review-resolution),  as most recently discussed in #2266 , implemented in #2346 and approved by the SC in python/steering-council#113:

> Except where otherwise approved by the Steering Council, pronouncements of PEP resolution will be posted to the [Python-Dev](https://mail.python.org/mailman3/lists/python-dev.python.org/) mailing list.